cdap-runtime-ext-dataproc/src/main/resources/gcp-dataproc.json

```diff
@@ -475,6 +475,15 @@
             "size": "medium"
           }
         },
+        {
+          "widget-type": "textbox",
+          "label": "Temp Bucket",
```

Comment:
I think we should change "GCS Bucket" (previous textbox) label to "Staging Bucket" to prevent confusion

Reply:
Done